addons/website_event_track/models/event_track.py

```diff
@@ -51,31 +52,44 @@ def _get_default_stage_id(self):
              " * Grey is the default situation\n"
              " * Red indicates something is preventing the progress of this track\n"
              " * Green indicates the track is ready to be pulled to the next stage")
-    # speaker
-    partner_id = fields.Many2one('res.partner', 'Speaker')
+    # partner_id is the partner used to contact the speaker.
+    partner_id = fields.Many2one('res.partner', 'Contact', help="partner_id is the partner used to contact the speaker.")
+    # these fields describe the public speaker personal information.
     partner_name = fields.Char(
-        string='Name', compute='_compute_partner_info',
-        readonly=False, store=True, tracking=10)
+        string='Name', compute='_compute_partner_name',
+        readonly=False, store=True, tracking=10,
+        help='The value could be different from partner_id.name as this is speaker public information')
```

Comment:
Help should be targeting end users, so maybe something more like "Name is used for public display and may vary from contact name."